configs/nativescript-vue.json

```diff
@@ -17,19 +18,20 @@
   "stop_urls": [],
   "selectors": {
     "lvl0": {
-      "selector": "//li[contains(@class, 'level-0')]//a[contains(@class, 'docSearch-active')][1]/preceding::span[1]",
+      "selector": "selector": "//li[contains(@class, 'level-0')]//a[contains(@class, 'docSearch-active')][1]/preceding::span[1]",
```

Comment:
Mistake here

Reply:
Ah yes thanks! Tinkered with it and left it like that accidentally!